sharepoint-adfs/azuredeploy.json

```diff
@@ -259,54 +275,155 @@
       "script": "ConfigureSPVM.ps1",
       "function": "ConfigureSPVM",
       "forceUpdateTag": "1.0"
+    },
+    "dscConfigureFEVM": {
+      "fileName": "ConfigureFEVM.zip",
+      "script": "ConfigureFEVM.ps1",
+      "function": "ConfigureFEVM",
+      "forceUpdateTag": "1.0"
     }
   },
   "resources": [
     {
-      "name": "[variables('generalSettings').storageAccountName]",
-      "type": "Microsoft.Storage/storageAccounts",
+      "apiVersion": "2015-06-15",
```

Comment:
let's update these apiVersions - I realize they won't match across the template (unless you want to go hit those as well :))

Reply:
Ok, I updated all apiVersions :)